Makefile mkdir command does not work on MacOS

### Version Check

- [x] I have updated to `v2.0.0-beta.9` or later.
- [x] I have updated to the `nightly` build from last night (or latest one available).
- [x] I have tested with the latest source code from `main` as of today.

### What browser are you using?

n/a

### How are you loading Kitten Scientists?

None

### Describe the Bug

The Makefile uses mkdir --parents option.
MacOS mkdir command does not understand the --parents option

```
➜  kitten-scientists git:(main) make
yarn vite --config vite.config.inject.js build
vite v6.2.2 building for production...
✓ 730 modules transformed.
output/kitten-scientists.inject.js  2,027.78 kB │ gzip: 425.32 kB │ map: 1,067.11 kB
output/kitten-scientists.inject.js  2,063.12 kB │ gzip: 426.40 kB │ map: 1,069.76 kB
✓ built in 816ms
mkdir --parents devcontainer/overlay/ && cp output/kitten-scientists.inject.js devcontainer/overlay/kitten-scientists.inject.js
mkdir: illegal option -- -
usage: mkdir [-pv] [-m mode] directory_name ...
make: *** [injectable] Error 64
➜  kitten-scientists git:(main)
```

changing the mkdir --parents to mkdir -p fixes the issue on Macintosh and should still be compatible with linux

```
➜  kitten-scientists git:(mac-compatibility) make
yarn vite --config vite.config.inject.js build
vite v6.2.2 building for production...
✓ 730 modules transformed.
output/kitten-scientists.inject.js  2,027.78 kB │ gzip: 425.32 kB │ map: 1,067.11 kB
output/kitten-scientists.inject.js  2,063.12 kB │ gzip: 426.40 kB │ map: 1,069.76 kB
✓ built in 821ms
mkdir -p devcontainer/overlay/ && cp output/kitten-scientists.inject.js devcontainer/overlay/kitten-scientists.inject.js
node devcontainer/build-devcontainer.js
➜  kitten-scientists git:(mac-compatibility)
```

### Your Savegame

_No response_

### To Reproduce

_No response_

### Screenshots

_No response_

--- a/Makefile
+++ b/Makefile
@@ -48,7 +48,7 @@ output: node_modules
 .PHONY: injectable
 injectable: node_modules
 	yarn vite --config vite.config.inject.js build
-	mkdir --parents devcontainer/overlay/ && cp output/kitten-scientists.inject.js devcontainer/overlay/kitten-scientists.inject.js
+	mkdir -p devcontainer/overlay/ && cp output/kitten-scientists.inject.js devcontainer/overlay/kitten-scientists.inject.js
 
 .PHONY: userscript
 userscript: node_modules

